frontend/src/App.js

```diff
@@ -35,8 +35,9 @@ function App() {
               <Route path="/home" element={<HomePage />} />
               <Route path="/profile" element={<ProfilePage />} />
               <Route path="/match" element={<MatchPage />} />
+              {/* Todo: Instead of using URL parameter, pass a room id prop */}
+              <Route path="/room/:roomId" element={<RoomPage />} />

```

Comment:
With reference to [PR#33](https://github.com/CS3219-AY2223S1/cs3219-project-ay2223s1-g28/pull/33), since `roomId` is returned from the matching service upon successful match and passed as a state object when navigating to the room page, you might wanna adjust the route here and the logic in `Room/index.js`.